src/modules/json.wren

```diff
@@ -52,7 +53,10 @@ class Token {
 }
 
 class JSONStream {
-  foreign stream_begin(value)
+  // Ensure the stream is always a string
+  stream_begin(value){stream_begin_(value.toString)}
```

Comment:
I was thinking more of an assertion like we have elsewhere since so many things can be coerced to strings that this "blanket" approach always scares me. IE:

```
  // TODO: Copied from `io`. Figure out good way to share this.
   static ensureString_(s, name) {
     if (!(s is String)) Fiber.abort("%(name) must be a string.")
   }
```

And our variable name here is probably `json` or `jsonText`, yes?

Reply:
In that case is better to import the `Assert` lib to check that.

Reply:
OTTOMH I feel they serve different purposes - but let me go look at it and think on it.  The long-term plan is to have an `Ensure` module that's shared across things - likely at the `wren-console` layer since we could use it for CLI stuff also.